lib/ssrf_filter/ssrf_filter.rb

```diff
@@ -131,7 +131,7 @@ class CRLFInjection < Error
   end
 
   def self.unsafe_url?(url)
-    uri = URI(url)
+    uri = Addressable::URI.parse(url)
```

Comment:
have you looked at the other use of `URI(url)` on line 117? i think `SsrfFilter.get` is only used in pagegrab.rb but it might get used in other places in future?

Reply:
i looked into also changing L117 but `Addressable::URI` returns `port` as `nil` by default, whereas URI() doesn't

I did a few fixes for `port` but tests for SNI, TLS certificates fail amongst others, and I don't see (yet?) a fix

Reply:
@heri `unsafe_url?` is our predominant use case, so i'm good with merging this and treat the remaining stuff as a future enhancement.